src/components/MusicPlayer.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {useState} from 'react'
+import { BiSkipPrevious, BiSkipNext, BiPause, BiPlay } from 'react-icons/bi';
+import { songs } from '../assets/items';
+
+export default function MusicPlayer(props: any) {
+  const [isPlaying, setIsPlaying] = useState(true)
+  const [isPlayAnime, setIsPlayAnime] = useState(false)
```

Comment:
looks like a redundant state. Why can't we govern the same behaviour with `isPlaying` state?